### PR TITLE
Animation Under All Tabs are Working

### LINKED ISSUE #295: Animation effects of Tabs only change while going back from ABOUT section though the page changes
refer https://github.com/coala/projects/issues/179
while changing tabs from about to others and then coming back through back button on browser works fine but if we see other section except ABOUT  then same problem is there as in refered issue

### PR DESCRIPTION
**Pull request by AkshJain99**
Highlighting animation are working
under all tabs on navigation
between different tabs.

Fixes https://github.com/coala/landing-frontend/issues/295

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Review by naveentvelu — COMMENTED** [review]
Adding ul.tabs in partials/tabs/people.html is not desired. Because only five tabs are defined in index.html. And /people tab is not defined there. It would be better if we remove it.

**Comment by AkshJain99** [comment]
ya but people.html is somewhere which also have this problem so maybe it is right 
I will look again

**Comment by AkshJain99** [comment]
@hemangsk  please revirew this pr

**Comment by AkshJain99** [comment]
@li-boxuan  please take look at this pr

**Comment by hemangsk** [comment]
Is there any way to avoid the code duplication, maybe adding it by in index.html file ?

**Comment by AkshJain99** [comment]
@hemangsk  I tried to do so in index.html but I dont think that we will be able to do that .
Do we?
And its just 2 line of code that has been added I dont think that it is affecting the codes,Is it?
 :sweat_smile:

**Comment by AkshJain99** [comment]
@hemangsk  I tried adding the script in index.html but then it doesn't work, but code which I committed is working well :sweat_smile: :

File: partials/tabs/people.html [hunk 1]
```diff
@@ -35,3 +35,9 @@ <h4 class="no-margin light">{{ person.nick }}</h4>
   </div>
 </div>
 <link rel="stylesheet" href="/resources/css/people.css">
+<script>
+  $(document).ready(function(){
+    $('ul.tabs').tabs();
+  });
+
```
Comment (hemangsk) [inline partials/tabs/people.html]: Extra line, can be removed

Comment (hemangsk) [reply partials/tabs/people.html]: Same for all the files

**Comment by hemangsk** [comment]
Ya I understand, Two lines or one lines is not the point, duplication makes maintaining it difficult. Otherwise, this lgtm.

File: partials/tabs/coalaonline.html [hunk 2]
```diff
@@ -274,3 +274,9 @@ <h6 class="uppercase"><strong><span class="lowercase">c</span>OAFILE</strong></h
     </div>
   </div>
 </div>
+<script>
+$(document).ready(function(){
+  $('ul.tabs').tabs();
+});
+
+</script>
```
Comment (jayvdb) [inline partials/tabs/coalaonline.html]: Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp8v872ato/partials/tabs/coalaonline.html
+++ b/tmp/tmp8v872ato/partials/tabs/coalaonline.html
@@ -279,4 +279,4 @@
   $('ul.tabs').tabs();
 });
 
-</script>+</script>
```

File: partials/tabs/languages.html [hunk 3]
```diff
@@ -219,4 +220,4 @@
     $('.tooltipped').tooltip({delay: 50});
 
     })
-</script>
+</script>
```
Comment (jayvdb) [inline partials/tabs/languages.html]: Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp8v872ato/partials/tabs/languages.html
+++ b/tmp/tmp8v872ato/partials/tabs/languages.html
@@ -220,4 +220,4 @@
     $('.tooltipped').tooltip({delay: 50});
 
     })
-</script>+</script>
```

File: partials/tabs/home.html [hunk 4]
```diff
@@ -131,3 +131,9 @@ <h2 class="fine">get started</div>
         </div>
     </div>
 </section>
+<script>
+$(document).ready(function(){
+  $('ul.tabs').tabs();
+});
+
+</script>
```
Comment (jayvdb) [inline partials/tabs/home.html]: Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp8v872ato/partials/tabs/home.html
+++ b/tmp/tmp8v872ato/partials/tabs/home.html
@@ -136,4 +136,4 @@
   $('ul.tabs').tabs();
 });
 
-</script>+</script>
```

File: partials/tabs/about.html [hunk 5]
```diff
@@ -129,4 +129,4 @@ <h2 class="fine center">about</h2>
   $('ul.tabs').tabs();
 });
 
-</script>
+</script>
```
Comment (jayvdb) [inline partials/tabs/about.html]: Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp8v872ato/partials/tabs/about.html
+++ b/tmp/tmp8v872ato/partials/tabs/about.html
@@ -129,4 +129,4 @@
   $('ul.tabs').tabs();
 });
 
-</script>+</script>
```

File: partials/tabs/getinvolved.html [hunk 6]
```diff
@@ -77,3 +77,9 @@ <h4 class="light center">We’d love to share a bar of chocolate with you!
     </div>
 </div>
 </section>
+<script>
+ $(document).ready(function(){
+    $('ul.tabs').tabs();
+});
+
+</script>
```
Comment (jayvdb) [inline partials/tabs/getinvolved.html]: Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp8v872ato/partials/tabs/getinvolved.html
+++ b/tmp/tmp8v872ato/partials/tabs/getinvolved.html
@@ -82,4 +82,4 @@
     $('ul.tabs').tabs();
 });
 
-</script>+</script>
```

File: partials/tabs/people.html (hunk 1 above)
Comment (hemangsk) [reply partials/tabs/people.html]: Not the newline at end of file, See L42 , L281 etc.

Comment (AkshJain99) [reply partials/tabs/people.html]: done @hemangsk :+1:

Comment (hemangsk) [reply partials/tabs/people.html]: Sorry I think I wasn't clear enough, only those extra new lines which were added in this commit inside the script tags need to be omitted. Others are fine. If you feel they are extra then its a discussion for a new ticket and not this one.

So we are fine with just converting the
```
<script>
$(document).ready(function(){
 $('ul tabs').tabs();
})

``` 

to 

```
<script>
$(document).ready(function(){
 $('ul tabs').tabs();
})
```

Comment (AkshJain99) [reply partials/tabs/people.html]: oh okay @hemangsk done :+1:

**Comment by hemangsk** [comment]
ack 88f354e

**Comment by AkshJain99** [comment]
thanks @

**Comment by Makman2** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**